src/main/java/org/jenkinsci/plugins/gitlab/GitlabBuilds.java

```diff
@@ -49,7 +54,12 @@ public String build(GitlabCause cause, Map<String, String> customParameters, Git
     	if (hasCommitStatus(project, cause.getLastCommitId(), api)) {
         	shouldRun = false;
         }
-    	
+
+        if (lastNote != null && lastNote.getBody().equals(triggerComment)) {
+            LOGGER.info("Trigger comment found");
+            shouldRun = true;
```

Comment:
i did exactly the thing as your about a month ago, then i gave up :)
i think this code snippet will trigger the building `forever`?